### DISCUSSION TITLE
I want to increase the height of the plot output of an ojs chunk in a quarto document. How do I do this?

### DISCUSSION THREAD

**Discussion by j-kind** [quarto-dev/quarto-cli#6750] [Q&A]
### Description

I'm using ojs to generate an echarts network graph. I would like to increase the height of the output so that I can show a larger network without the need to zoom in and out.  Is there a setting that would allow this?
I've tried:

//| fig-height: 6 
//| height: 6
//| out-height: 6

And setting the same globally.

**Comment by cscheid** [comment]
> I'm using ojs to generate an echarts network graph.

Since we don't support echarts natively, you'll need to use their API to generate a larger SVG file directly.
